components/Header.js

```diff
@@ -74,11 +74,11 @@ export default function Header({ navigation, backButton, homeDisabled }) {
               <PagesLogo width={logoWidth} height={logoHeight} />
             </View>
           ) : (
-            <TouchableWithoutFeedback onPress={toMainMenu}>
+            <TouchableOpacity onPress={toMainMenu}>
```

Comment:
This was causing weird artifacts on touch, so I replaced it with <TouchableWithoutFeedback>

Let's double check this, and if it works fine this time, I'd agree on having it back